pkg/sdk/testint/network_policies_gen_integration_test.go

```diff
@@ -85,6 +108,90 @@ func TestInt_NetworkPolicies(t *testing.T) {
 		assert.Equal(t, 1, np.EntriesInBlockedIpList)
 	})
 
+	t.Run("Alter - set allowed network rule list", func(t *testing.T) {
```

Comment:
This test is failing on the CI:` FAIL: TestInt_NetworkPolicies/Alter_-_set_allowed_network_rule_list (2.05s)`.

Reply:
fixed